src/Static.ts

```diff
@@ -8,18 +8,37 @@ export class Static {
     }
 
     public static buildSectionBlock(mrkdwnValue: string): SectionBlock {
-        return { type: "section", text: { type: "mrkdwn", text: mrkdwnValue } };
+        return {type: "section", text: {type: "mrkdwn", text: mrkdwnValue}};
     }
 
     public static buildContextBlock(mrkdwnValue: string): ContextBlock {
-        return { type: "context", elements: [ { type: "mrkdwn", text: mrkdwnValue } ] };
+        return {type: "context", elements: [{type: "mrkdwn", text: mrkdwnValue}]};
     }
 
     public static buildSelectOption(text: string, value: string): Option {
-        return { text: this.buildTextElem(text), value: value };
+        return {text: this.buildTextElem(text), value: value};
     }
 
     public static buildTextElem(text: string): PlainTextElement {
-        return { type: "plain_text", text, emoji: true };
+        return {type: "plain_text", text, emoji: true};
+    }
+    public static buildVoteOptions(parameters: string [], start: number ): ActionsBlock[] {
+        const actionBlocks: ActionsBlock[] = [{ type: "actions", elements: [] }];
+        let actionBlockCount = 0;
+        // Construct all the buttons
+        for (let i = start; i < parameters.length; i++) {
+            if (i % 5 === 0 && i != 0) {
+                const newActionBlock: ActionsBlock = { type: "actions", elements: [] };
+                actionBlocks.push(newActionBlock);
+                actionBlockCount++;
+            }
+            // Remove special characters, should be able to remove this once slack figures itself out
+            parameters[i] = parameters[i].replace("&amp;", "+").replace("&lt;", "greater than ")
```

Comment:
You are replacing &lt; with greater than and &gt; with less than. It's the other way around I believe.

Reply:
From my understanding, we are removing the special characters, not adding them.  This line is directly from previous versions.

Reply:
I realized just now it's from the previous versions. I believe we messed up and lt should be less than not greater than and vice versa.

Reply:
Okay I'll fix it

Reply:
Fixed